exampleCourse/courseInstances/SectionA/__docs/index.md

```diff
@@ -0,0 +1,32 @@
+## Demo Course (XC 100)
+
+This [demo course](https://www.prairielearn.org/pl/course_instance/128605) was created to highlight the PrairieLearn features which allow instructors to create flexible, authentic and robust online assessments, including formative, summative, individual and group work.
+
+<!--### Pre-Lecture Activities
+
+You can use PrairieLearn assessments to provide a collection of lecture notes, slides, videos, JupyterLabs, and other course resources, combined with short quiz questions to help students check their learning progress.
+
+Check the demo pre-lecture assessment [L1: Presenting course content with short quiz questions](). More details regarding this assessment [here](../assessments/03-LectureContent/__docs/docs.md).-->
```

Comment:
What are these commented-out lines for? Are these being used somewhere? If not, can we just delete them?

Reply:
I did not touch these documentations, since I am still working on the marketing website. I actually think some of the files (for example this one) should be completely removed from the exampleCourse, and moved to marketing.

Reply:
we decided to leave these comments "as is" for now